package.json

```diff
@@ -165,8 +165,8 @@
     "json": "^11.0.0",
     "lint-staged": "^12.3.4",
     "prettier": "^2.5.1",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
+    "react": ">=17.0.2",
+    "react-dom": ">=17.0.2",
```

Comment:
Does this help your issue?
I wonder if changing peer dependency version somehow would help.

Reply:
https://github.com/pmndrs/react-three-fiber/issues/2171
Alright, it's a DEV only issue. I don't think it's rational to change the dev dependency with range. Would be nice to find a better solution.